proposals/templates/proposals/proposal_submit.html

```diff
@@ -121,7 +121,12 @@ <h2>
                 </div>
             {% endif %}{% csrf_token %}
                 <table>{{ form.as_table }}</table>
-                {% if troublesome_pages %}
+                {% if is_practice %}
+                <p>
+                    {% trans "U bewerkt op het moment een oefenstudie. Deze kan niet ter beoordeling door de FETC-GW worden ingediend." %}
```

Comment:
It might be better to wrap this is a ``<div class="info"`` for better visibility